hs_err_pid71087.log

```diff
@@ -0,0 +1,639 @@
+#
+# A fatal error has been detected by the Java Runtime Environment:
+#
+#  SIGSEGV (0xb) at pc=0x0000000114b5879f, pid=71087, tid=0x0000000000005903
+#
+# JRE version: Java(TM) SE Runtime Environment (8.0_161-b12) (build 1.8.0_161-b12)
+# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.161-b12 mixed mode bsd-amd64 compressed oops)
+# Problematic frame:
+# J 9872 C2 java.io.File.<init>(Ljava/lang/String;)V (47 bytes) @ 0x0000000114b5879f [0x0000000114b58760+0x3f]
+#
+# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
+#
+# If you would like to submit a bug report, please visit:
+#   http://bugreport.java.com/bugreport/crash.jsp
+#
+
+---------------  T H R E A D  ---------------
```

Comment:
Please remove this file. JVM errors, we shouldn't have something like this in codebase

Reply:
:man_facepalming:

Reply:
Done! :)